public/app/core/components/PageHeader/PageHeader.tsx

```diff
@@ -10,7 +10,12 @@ export interface Props {
 }
 
 const SelectNav = ({ main, customCss }: { main: NavModelItem; customCss: string }) => {
-  const defaultSelectedItem = main.children.find(navItem => {
+  const { children } = main;
+  if (!children || children.length === 0) {
+    return null;
```

Comment:
This also changes the return type.

Reply:
@aocenas This one doesn't have a return type set strictly, it's just typed props destructuring in an arrow function - it doesn't render anything if children doesn't exist or is an empty array.

Reply:
Ah right was reading that wrong.